src/index.ts

```diff
@@ -278,6 +291,16 @@ export function createBirpc<RemoteFunctions = Record<string, never>, LocalFuncti
     off(onMessage)
   }
 
+  function rejectPendingCalls(handler: PendingCallHandler) {
```

Comment:
The naming seems confusing because you are not required to reject it, it just calls a function on every entry